blog/2024-06-03-ratify-1.2.0.mdx

```diff
@@ -0,0 +1,71 @@
+
+## A brand new Cosign experience! 
+
+Starting in v1.2.0, Ratify now has more comprehensive Cosign verification capabilities. 
+
+Verification of Cosign signatures is a critical validation. As such, the Cosign verifier is now a built-in verifier along with the Notation verifier. Beyond first class support, the Cosign verifier introduces a `Trust Policy` which binds a particular set of verification settings to a certain registry scope. Now, it's possible to specify multiple trust policies based on your registry, namespace, and/or image. 
+
+Cosign verification also includes a robust key experience that integrates directly with KMP resources. Expanded capabilities include: multiple key support from KMP or filesystem and RSA + elliptical keys across multiple sizes.
+
+<!-- TODO: update links -->
+
+Watch this short demo to see the Cosign Verifier in action. Or check out the [documentation](https://ratify.dev/docs/reference/verifiers/cosign) for more information.
+
+[![asciicast](https://asciinema.org/a/658139.svg)](https://asciinema.org/a/658139)
+
+## Support for Multi-Tenancy in Kubernetes
+
+Ratify now supports multi-tenancy, enabling organizations to share a single cluster among multiple teams by allowing users to define resources within a namespace scope. 
+
+This enhancement is seamlessly integrated, as users implicitly opt into multi-tenancy by applying namespace-specific resources such as NamespacedPolicy and NamespacedVerifier, without the need for an explicit feature flag. 
+
+Currenlty multi-tenancy supports the following resources: `Referrer Store`, `Verifier`, and `Key Management Provider`, allowing these to be defined and managed within specific namespaces. 
+
+Additionally, cluster-wide resources can be set as defaults for namespaces that do not have specific resources defined, ensuring flexibility and efficient resource management across the organization.
+
+<!-- TODO: update links -->
+Watch this step-by-step tutorial to see multi-tenancy in action. Or check out the [documentation](https://ratify.dev/docs/reference/multi-tenancy) for more information.
+
+[![asciicast](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj.svg)](https://asciinema.org/a/PO8RPcuZ2qyShif80c55WQipj)
+
+## Introduction of the Key Management Provider
+
+In this release, we are introducing the KeyManagementProvider (KMP), a new resource designed to replace the existing CertificateStore. 
+
+The KMP resource is a comprehensive solution for managing and retrieving keys and certificates. It includes an inline provider, which allows for the specification of a single key or certificate. Additionally, it features an Azure Key Vault provider, which enables the fetching of any number of keys or certificates. 
```

Comment:
@akashsinghal I think the inline provider also allows for multiple keys or certificates, right?

Reply:
Good point. I revised the KPM section and remove that bit. Lmk what you think. :)